TECHMANIA/ProjectSettings/ProjectSettings.asset

```diff
@@ -247,15 +247,15 @@ PlayerSettings:
   AndroidTargetArchitectures: 1
   AndroidSplashScreenScale: 0
   androidSplashScreen: {fileID: 0}
-  AndroidKeystoreName: 
-  AndroidKeyaliasName: 
+  AndroidKeystoreName: C:/Users/sam_nya/OneDrive/Keys/user.keystore
```

Comment:
Looks like this file is specific to you. Can we maybe include it in the repo?

Reply:
I removed my release key here. It can build fine with the debug key